From 2c895a775783b6383d8a4acc83312688c0c8c031 Mon Sep 17 00:00:00 2001
From: Sven van Haastregt <sven.vanhaastregt@arm.com>
Date: Thu, 19 May 2022 13:50:18 +0100
Subject: [PATCH 037/172] [NFC] Fix mangled names in test

`clang/lib/AST/ItaniumMangle.cpp` doesn't include any `_t` in the
mangled name.

(cherry picked from commit d06c92387d49488af574c650a57d4fcfba881115)
---
 test/transcoding/check_wo_qualifier.ll | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/test/transcoding/check_wo_qualifier.ll b/test/transcoding/check_wo_qualifier.ll
index cd4c52db..81f28454 100644
--- a/test/transcoding/check_wo_qualifier.ll
+++ b/test/transcoding/check_wo_qualifier.ll
@@ -26,9 +26,9 @@ target triple = "spir64-unknown-unknown"
 ; Function Attrs: nounwind
 define spir_kernel void @sample_kernel(%opencl.image2d_array_wo_t addrspace(1)* %input) #0 !kernel_arg_addr_space !1 !kernel_arg_access_qual !2 !kernel_arg_type !3 !kernel_arg_base_type !5 !kernel_arg_type_qual !4 {
 entry:
-  %call.tmp1 = call spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo_t(%opencl.image2d_array_wo_t addrspace(1)* %input)
+  %call.tmp1 = call spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo(%opencl.image2d_array_wo_t addrspace(1)* %input)
   %call.tmp2 = shufflevector <2 x i32> %call.tmp1, <2 x i32> undef, <3 x i32> <i32 0, i32 1, i32 2>
-  %call.tmp3 = call spir_func i64 @_Z20get_image_array_size20ocl_image2d_array_wo_t(%opencl.image2d_array_wo_t addrspace(1)* %input)
+  %call.tmp3 = call spir_func i64 @_Z20get_image_array_size20ocl_image2d_array_wo(%opencl.image2d_array_wo_t addrspace(1)* %input)
   %call.tmp4 = trunc i64 %call.tmp3 to i32
   %call.tmp5 = insertelement <3 x i32> %call.tmp2, i32 %call.tmp4, i32 2
   %call.old = extractelement <3 x i32> %call.tmp5, i32 0
@@ -36,10 +36,10 @@ entry:
 }
 
 ; Function Attrs: nounwind
-declare spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo_t(%opencl.image2d_array_wo_t addrspace(1)*) #0
+declare spir_func <2 x i32> @_Z13get_image_dim20ocl_image2d_array_wo(%opencl.image2d_array_wo_t addrspace(1)*) #0
 
 ; Function Attrs: nounwind
-declare spir_func i64 @_Z20get_image_array_size20ocl_image2d_array_wo_t(%opencl.image2d_array_wo_t addrspace(1)*) #0
+declare spir_func i64 @_Z20get_image_array_size20ocl_image2d_array_wo(%opencl.image2d_array_wo_t addrspace(1)*) #0
 
 attributes #0 = { nounwind }
 
-- 
2.20.1

